src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -349,14 +349,14 @@ public function orWhereDoesntHaveMorph($relation, $types, Closure $callback = nu
     }
 
     /**
-     * Add subselect queries to include an aggregate value for a relationship.
+     * Add subselect queries to include aggregate values for relationships.
      *
      * @param  mixed  $relations
      * @param  string  $column
      * @param  string  $function
      * @return $this
      */
-    public function withAggregate($relations, $column, $function = null)
+    public function withAggregate($relations, $column = null, $function = null)
```

Comment:
This is a breaking change.

Reply:
@driesvints Thanks for looking into this. I'm sorry if I was on the wrong track here. I assumed making a required parameter optional would not be considered a breaking change. I think that nothing would actually break if it kept using the current syntax.

Reply:
https://3v4l.org/VNAms

Reply:
@driesvints Got it. Thank you very much for clarifying.

Reply:
@driesvints Should I change the title to [9.x]? Sorry, I just started contributing recently.

Reply:
You'll need to submit a new PR to the master branch for this.

Reply:
you need to update the phpdoc too